dotnet-interactive.Tests/KernelHttpApiTests.cs

```diff
@@ -76,35 +90,83 @@ public async Task variable_is_returned_with_application_json_content_type(Langua
         [InlineData(Language.FSharp)]
         public async Task When_variable_does_not_exist_then_it_returns_404(Language language)
         {
-            var response = await _server.HttpClient.GetAsync($"/variables/{language.LanguageName()}/does_not_exist");
+            var response = await GetServer(language).HttpClient.GetAsync($"/variables/{language.LanguageName()}/does_not_exist");
 
             response.StatusCode.Should().Be(HttpStatusCode.NotFound);
         }
 
         [Fact]
         public async Task When_subkernel_does_not_exist_then_it_returns_404()
         {
-            var response = await _server.HttpClient.GetAsync("/variables/does_not_exist/does_not_exist");
+            var response = await GetServer().HttpClient.GetAsync("/variables/does_not_exist/does_not_exist");
 
             response.StatusCode.Should().Be(HttpStatusCode.NotFound);
         }
 
+        [Theory]
+        [InlineData(Language.CSharp)]
+        public async Task lsp_textDocument_hover_returns_expected_placeholder(Language language)
+        {
+            var request = JObject.Parse(@"
+{
+    ""textDocument"": {
+        ""uri"": ""path/to/document""
+    },
+    ""position"": {
+        ""line"": 1,
+        ""character"": 2
+    }
+}
+");
+            var response = await GetServer(language).HttpClient.PostJsonAsync("/lsp/textDocument/hover", request);
+            await response.ShouldSucceed();
+            var responseJson = await response.Content.ReadAsStringAsync();
+            var json = JObject.Parse(responseJson);
+            var responseType = json["contents"]["kind"].Value<string>();
+            responseType.Should().Be("markdown");
```

Comment:
In cases where there are multiple assertions, you can make sure we get feedback on all of them by doing this before beginning the assertions:

```csharp
using var _ = new AssertionScope();
```